wcet/Makefile

```diff
@@ -0,0 +1,342 @@
+# Makefile to build and run the examples using KLEE.
```

Comment:
@rasoolmaghareh I've changed the PR title to better reflect the content of this PR.